### DISCUSSION TITLE
Usage of OIDCScrubRequestHeaders

### DISCUSSION THREAD

**Discussion by benoitsln** [OpenIDC/mod_auth_openidc#981] [Q&A] [answered]
I found this setting in the documentation (auth_openidc.conf, containing all settings): **OIDCScrubRequestHeaders**
But it seems to not exists, Apache does not accept it: _Invalid command 'OIDCScrubRequestHeaders', perhaps misspelled or defined by a module not included in the server configuration_
A search in the repository for this option return only the documentation and the #define variable: it seem not implemented (https://github.com/zmartzone/mod_auth_openidc/search?q=OIDCScrubRequestHeaders)
Is there any other way to debug ?
Thanks !

**Comment by zandbelt** [comment, marked as answer]
this option has been removed since version 2.4.0 (we'll also remove the leftover #define and config doc) because it was abusively used in some production deployments; the way to test is to disable mod_auth_openidc and set the headers you want to test from a browser plugin (or a tool like curl), thanks for the report
